ORBIT/phases/install/quayside_assembly_tow/moored.py

```diff
@@ -363,6 +480,11 @@ def install_moored_substructures(
         installation at site.
     """
 
+    warn(
```

Comment:
If this is functionally replaced elsewhere in the code, then we can just delete it because this is pretty internal-to-ORBIT code, so users won't interface with this change. Though, if I'm wrong, do let me know.